test/e2e/framework/helper/featureset/featureset.go

```diff
@@ -126,4 +126,8 @@ const (
 	// OnlySAN denotes whether the target issuer is able to sign certificates
 	// with only SANs set
 	OnlySAN = "OnlySAN"
+
+	// SaveCAToSecretFeature denotes whether the target issuer returns a CA
+	// certificate which can be stored in the CA.crt field of the Secret.
+	SaveCAToSecretFeature Feature = "SaveCAToSecretFeature"
```

Comment:
nit: Remove `Feature` type declaration, and remove "Feature" suffix in the string